src/modules/Permissions/components/ManagePermissionsDialog/CreatePermissions.tsx

```diff
@@ -56,6 +57,8 @@ export type CreatePermissionsProps = {
     | 'inline'
     | undefined;
   kafkaName: string | undefined;
+  isExpanded: boolean;
+  onChangeExpendedSection: (isExpanded: boolean) => void;
```

Comment:
Really minor nit here, but can you fix the typo and change this to `onChangeExpandedSection` (here and throughout)?